go.mod

```diff
@@ -5,9 +5,10 @@ require (
 	github.com/akutz/gournal v0.5.0
 	github.com/container-storage-interface/spec v1.5.0
 	github.com/cucumber/godog v0.10.0
+	github.com/dell/dell-csi-extensions/replication v1.0.0
 	github.com/dell/gocsi v1.5.0
 	github.com/dell/gofsutil v1.6.0
-	github.com/dell/goisilon v1.6.1-0.20220110083433-83d13e163ea4
+	github.com/dell/goisilon v1.5.0
```

Comment:
please update this to v1.6.0. Also update the commit version in replace directive for this module if required.